include/util/percent.hpp

```diff
@@ -71,6 +79,13 @@ class Percent
             {
                 std::cout << ".";
             }
+#ifdef WIN32
+            if (!_isatty(_fileno(stdout)))
+                std::cout << std::endl;
+#else
+            if (!isatty(fileno(stdout)))
+                std::cout << std::endl;
+#endif
```

Comment:
Instead of doing this here (and in other files in osrm-backend) what about a utility header providing a portable isatty function hiding all the platform specifics.